admin-ui/src/components/app-shell.tsx

```
@@ -29,8 +29,10 @@ function NavItem({
       to={to}
       className={({ isActive }) =>
         cn(
-          "text-sm font-medium transition-colors hover:text-foreground",
-          isActive ? "text-foreground" : "text-muted-foreground"
+          "font-display relative text-[0.7rem] uppercase tracking-[0.22em] transition-colors",
+          isActive
```

ESLint

[error] Insert `·?`

(prettier/prettier)